secure_storage/ta/secure_storage_ta.c

```diff
@@ -25,6 +25,8 @@
  * POSSIBILITY OF SUCH DAMAGE.
  */
 
+#include <inttypes.h>
+
```

Comment:
Empty line not needed.

Reply:
I used other examples as an example.
https://github.com/linaro-swg/optee_examples/blob/master/aes/ta/aes_ta.c#L27
https://github.com/linaro-swg/optee_examples/blob/master/acipher/ta/acipher_ta.c#L6

But here it is not like this
https://github.com/linaro-swg/optee_examples/blob/master/hotp/ta/hotp_ta.c#L7

Reply:
I'd say that it's incorrect there too then. Not that it urgently needs to be fixed, but I see no reason to let it spread.

Reply:
Done.